### PR TITLE
Add test for testing the delete method of uploads

### PR DESCRIPTION
**Pull request by lubosmj**
closes #5145
https://pulp.plan.io/issues/5145

### CODE REVIEW

File: pulpcore/tests/functional/api/test_crd_uploads.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    FILE_TO_BE_CHUNKED_URL,
+)
+
+
+class UploadsDeleteFileSystemTestCase(unittest.TestCase):
+    """Test a deletion of an uploaded content
+
+    This test targets the following issue:
+
+    * `Pulp #4186 <https://pulp.plan.io/issues/5092>`_
```
Comment (dkliban) [inline pulpcore/tests/functional/api/test_crd_uploads.py]: link and issue are not the same

Comment (kersommoura) [reply pulpcore/tests/functional/api/test_crd_uploads.py]: Good job. 

This test could be moved to the  `pulpcore/tests/functional/api/using_plugin/test_upload.py`, and reuse the code already there.

Comment (lubosmj) [reply pulpcore/tests/functional/api/test_crd_uploads.py]: I think that the test is not related to the plugin. Is it still necessary to move it to a different destination?

File: pulpcore/tests/functional/api/test_upload.py [hunk 2]
```diff
@@ -82,3 +84,74 @@ def test_create_artifact(self):
...
+class UploadsDeleteFileSystemTestCase(unittest.TestCase):
+    """Test a deletion of an uploaded content
+
+    This test targets the following issue:
+
+    * `Pulp #5092 <https://pulp.plan.io/issues/5092>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
```
Comment (dkliban) [inline pulpcore/tests/functional/api/test_upload.py]: This setUpClass method looks teh same as that of the ChunkedUploadTestCase class. You should just add the test_delete_upload() method to that class.

File: pulpcore/tests/functional/api/test_upload.py [hunk 3]
```diff
@@ -22,22 +24,56 @@
...
+        # fetch a name of the upload from the corresponding _href
+        upload_name = upload_request['_href'].replace('/pulp/api/v3/uploads/', '')[:-1]
+
+        self.addCleanup(self.client.delete, upload_request['_href'])
+        cmd = ('ls', os.path.join(MEDIA_PATH, 'upload', upload_name))
+        self.cli_client.run(cmd, sudo=True)
+
+        # delete
+        self.doCleanups()
+        with self.assertRaises(CalledProcessError):
```
Comment (dkliban) [inline pulpcore/tests/functional/api/test_upload.py]: What do you think about adding an assertion that you can perform an 'ls' before you do the delete?

Comment (dkliban) [reply pulpcore/tests/functional/api/test_upload.py]: I now see that you already do this on line 69.

**Pull request merged**